mod-inn-reach/src/main/resources/volaris/features/inn-reach-location.feature

```diff
@@ -13,23 +15,125 @@ Feature: Inn reach location
     * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenAdmin)', 'Accept': 'application/json'  }
 
     * configure headers = headersUser
+    * def notExistedLocationId = callonce uuid1
 
-  @Undefined
-  Scenario: Delete location
-    * print 'Delete location'
+  Scenario: Create inn reach location
+    * configure headers = headersUser
+    * print 'Create inn reach location 1'
+    Given path 'inn-reach/locations'
+    And request
+    """
+     {
+      "code": "scdes",
+      "description": "Steelcase Design Library"
+     }
+    """
+    When method POST
+    Then status 201
+    * def locationId1 = $.id
+
+    * print 'Get central server by id 1'
+    Given path '/inn-reach/locations', locationId1
+    When method GET
+    Then status 200
+
+    * def locationResponseOne = $
+    And match locationResponseOne.code == "scdes"
+    And match locationResponseOne.description == "Steelcase Design Library"
 
-  @Undefined
-  Scenario: Get location by id
-    * print 'Get location by id'
+    * print 'Create inn reach location 2'
+    Given path 'inn-reach/locations'
+    And request
+    """
+     {
+      "code": "plgen",
+      "description": "GVSU Pew Library General Collection"
+     }
+    """
+    When method POST
+    Then status 201
+    * def locationId2 = $.id
+
+    * print 'Get central server by id 2'
+    Given path '/inn-reach/locations', locationId2
+    When method GET
+    Then status 200
+
+    * def locationResponseTwo = $
+    And match locationResponseTwo.code == "plgen"
+    And match locationResponseTwo.description == "GVSU Pew Library General Collection"
 
-  @Undefined
   Scenario: Get locations
     * print 'Get locations'
+    * configure headers = headersUser
+    Given path '/inn-reach/locations'
+    When method GET
+    Then status 200
+    And match response.totalRecords == 2
 
-  @Undefined
-  Scenario: Create inn reach location
-    * print 'Create inn reach location'
+  Scenario: Check not existed location
+    * configure headers = headersUser
+    * print 'Check not existed location'
+    Given path '/inn-reach/locations', notExistedLocationId
+    When method GET
+    Then status 404
 
-  @Undefined
   Scenario: Update location
+    * configure headers = headersUser
     * print 'Update location'
+    Given path 'inn-reach/locations'
+    And request
+    """
+     {
+      "code": "ongen",
+      "description": "GVSU Steelcase Library General Collection"
+     }
+    """
+    When method POST
```

Comment:
POST method for updating?
should be PUT and Response 204 according to https://s3.amazonaws.com/foliodocs/api/mod-inn-reach/s/settings.html#operation/updateLocation